docs/source/installation_quickstart.rst

```diff
@@ -0,0 +1,134 @@
+.. _installation:
+
+============
+Install pyiron on your cluster in 10 minutes!
+============
+
+1. First, fetch mambaforge (mamba is essentially faster conda) install script from the web:
+
+.. code-block:: bash  
+    wget https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Linux-x86_64.sh
+
+2. Execute the script to install mambaforge:
+.. code-block:: bash
+    bash Mambaforge-Linux-x86_64.sh
+
+.. note:: 
+    NOTE: There are systems with a very tight filequota/memory quota on the home directory. In that case, you may need to install on a different directory. Usually there is a /software or /group directory that users can have permanent storage for software on. 
+
+3. Refresh/restart your shell:
+.. code-block:: bash
+    source .bashrc
+
+4. Now you have the option of installing pyiron in an environment with:
+.. code-block:: bash
+    mamba create -n YOURENVNAME
+Change `YOURENVNAME` to your liking.
+
+5. Then activate your environment with:
+.. code-block:: bash
+    mamba activate YOURENVNAME
+
+6. Call this to install pyiron:
+.. code-block:: bash
+    mamba install -c conda-forge pyiron pyiron_contrib
+
+This can take some time, so just hang tight.
+
+7. Now, we create a `pyiron_resources` folder. This can be placed anywhere, but here we place it in our home folder (e.g. `/home/abc123`).
+You can figure out the absolute path of your home directory is by calling `echo $HOME`
+.. code-block:: bash
+    mkdir /home/abc123/pyiron_resources
+
+8. Now, create our pyiron configuration file, `.pyiron` in the home folder. Paste the following lines into the file:
+.. code-block:: bash
+    [DEFAULT]
+    RESOURCE_PATHS = /home/abc123/pyiron_resources, /software/abc123/mambaforge/envs/pyiron/share/pyiron
+    PROJECT_CHECK_ENABLED = False
+    #DISABLE_DATABASE = True
+    FILE = ~/pyiron.db
+```
```

Comment:
Is thin not already automated using the install_dialog function?

And pyiron should automatically add the approptiate directories in the `$CONDAHOME` to the resources. From there on you could go on to configure :)

Reply:
In my experience (installed like 5x over the last week), a fresh install from mamba according to the instructions up to this point does not trigger an automatic generation of the `.pyiron` file. It does require the user to write the file in the home directory.

Reply:
Is the `install_dialog` function triggered upon loading pyiron modules for the first time? Or is it something that needs to be triggered on the python side? At any rate I don't think it does the user harm to know about the `.pyiron` configuration file even at the beginning.